content/automate/ManageIQ/Infrastructure/VM/Migrate/Email.class/__methods__/vmmigratetask_complete.rb

```diff
@@ -27,25 +27,30 @@
 $evm.log("info", "VM Owner: #{owner.inspect}")
 
 # to_email_address from owner.email then from model if nil
-to = owner.email || $evm.object['to_email_address']
+to = owner.email unless owner.nil?
```

Comment:
@billfitzgerald0120 Keep the original format and use `try` to handle the `nil` owner object.

`to = owner.try(:email) || $evm.object['to_email_address']`